app/src/main/java/com/example/guineapics/dao/GuineaPigsDAO.java

```diff
@@ -6,6 +6,18 @@
 
 public class GuineaPigsDAO implements DataAccessable<GuineaPig> {
```

Comment:
why is this a DAO? will you store these in a database?

Reply:
yeah, I'd like to switch to a database when I'm finished with implementing interactions